assets/files.wxs

```diff
@@ -1930,6 +1930,11 @@
             <File Id="filFF877991123D49A58FD15CB83AD44BA7" KeyPath="yes" Source="$(env.ProductSourcePath)\Modules\ThreadJob\ThreadJob.dll" />
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-linux`
Test-Connection.Error occurred in Describe block
```powershell
Exception calling "GetHostEntry" with "1" argument(s): "Name or service not known"
At /home/vsts/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1:24 char:9
```